apps/dc_tools/odc/apps/dc_tools/fs_to_dc.py

```diff
@@ -64,8 +62,8 @@ def cli(input_directory, update_if_exists, allow_unsafe, stac, statsd_setting, g
     for in_file in files_to_process:
         with in_file.open() as f:
             try:
-                if in_file.endswith(".yml") or in_file.endswith(".yaml"):
-                    metadata = yaml.safe_load(f, Loader=Loader)
```

Comment:
It is almost definitely worth keeping the `CLoader` for YAML parsing. For bulk loading it makes a big difference, just depends on where this tool is used.

Reply:
I would agree but `yaml.safe_load` doesn't take a Loader parameter - only `yaml.load` does.